pkg/xray/resource_xray_workers_count_test.go

```diff
@@ -0,0 +1,54 @@
+package xray
+
+import (
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/jfrog/terraform-provider-shared/util"
+)
+
+func TestAccWorkersCount_create(t *testing.T) {
```

Comment:
You test that create fails, but not delete. And you don't test that update works

Reply:
WOW! Ok. I guess we're just gonna assume it's all good. Sorry for not reading the description fully